src/main/resources/org/jenkinsci/plugins/GithubLogoutAction/index.jelly

```diff
@@ -0,0 +1,42 @@
+<!--
+ The MIT License
+ 
+ Copyright (c) 2016 CloudBees, Inc., James Nord
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in
+ all copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ THE SOFTWARE.
+ */
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+  <l:layout norefresh="true">
+    <l:main-panel>
+      <j:choose>
+        <j:when test="${h.isAnonymous()}">
+          <p>You are now logged out of Jenkins, however this has not logged you out of <a href="${it.gitHubURL}">GitHub</a>.</p>
```

Comment:
Or even better, add a method to `GithubLogoutAction.java` such as:

```java
    @Restricted(NoExternalUse.class) // jelly only
    public String getGitHubText() {
        SecurityRealm r = Jenkins.getInstance().getSecurityRealm();
        if (r instanceof GithubSecurityRealm) {
            GithubSecurityRealm ghsr = (GithubSecurityRealm) r;
            return (ghsr.getGithubWebUri() == ghsr.DEFAULT_WEB_URI)? "GitHub" : "GitHub Enterprise";
        }   
        // only called from the Jelly if the GithubSecurityRealm is set...
        return ""; 
    }   
```

This way in the Jelly config it can be referenced as:

```xml
<p>You are now logged out of Jenkins, however this has not logged you out of <a href="${it.gitHubURL}">${it.gitHubText}</a>.</p>
```